plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/ArtifactRetriever.java

```diff
@@ -146,30 +120,32 @@ public ArtifactRetriever() {
   }
 
   /**
-   * Returns the latest published artifact version, or null if there is no such version.
+   * Returns the latest published non-beta artifact version, or null if there is no such version.
```

Comment:
In this case where you consider the entire version range, I think the correct doc is something like below, unless you did not intend it.

 "The latest published non-beta artifact version or the latest published beta version if there all versions are beta. Otherwise null (i.e., wrong groupId or artifactId)." I was a bit confused before I looked into the `getLatestIncrementalVersion()` code.